ui-spacetimechart/src/components/TimeCaptions.tsx

```
@@ -112,16 +86,16 @@ const TimeCaptions: FC = () => {
       });
 
       // Render caption background:
-      ctx.fillStyle = 'white';
+      ctx.fillStyle = background;
       if (!swapAxis) {
         ctx.fillRect(0, spaceAxisSize - CAPTION_SIZE, timeAxisSize, CAPTION_SIZE);
       } else {
         ctx.fillRect(0, 0, CAPTION_SIZE, timeAxisSize);
       }
 
       // Render caption top border:
-      ctx.strokeStyle = '#979797';
-      ctx.lineWidth = 2;
+      ctx.strokeStyle = timeGraduationsStyles[1].color;
+      ctx.lineWidth = timeGraduationsStyles[1].width;
```

What if `timeGraduationsStyles.length === 1` ? Should we throw an error or use the default theme ?